dashboard/app/controllers/weblab_host_controller.rb

```diff
@@ -1,11 +1,17 @@
 class WeblabHostController < ApplicationController
-  layout false
-
   BRAMBLE_URL = 'https://downloads.computinginthecore.org/bramble_0.1.31'
   BRAMBLE_LOCALHOST_URL = 'http://127.0.0.1:8000/src'
+  STUDIO_URL = CDO.studio_url('', CDO.default_scheme)
 
   def index
-    @dev_mode = false; # Change to true to point to Bramble running on localhost
+    @dev_mode = false # Change to true to point to Bramble running on localhost
     @bramble_base_url = @dev_mode ? BRAMBLE_LOCALHOST_URL : BRAMBLE_URL
+    @studio_url = STUDIO_URL
+    @blank_load = params[:blank_load] == "true" # Special case for checking that Bramble can be initialized without loading files
```

Comment:
Open to naming ideas for this "blank load" case!

Reply:
hmmm, blank_load is pretty good, or maybe connection_test (could be too specific)?

Reply:
Maybe `@no_files` or `@skip_files` or `@skip_loading_files`?  The comment gives some naming inspiration :)

Reply:
Replaced with `skip_files`!